.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Add files/folders that should be ignored in .eslintignore.

Reply:
Added